cmd/checksum.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/cuducos/minha-receita/checksum"
+	"github.com/spf13/cobra"
+)
+
+var checksumTargetDir string
+
+const checksumHelper = `
+Checksum of the downloaded files.
+
+Even though the official website of the Brazilian Federal Revenue does not offer 
+a checksum for their files, this command can be used to create or check the checksum
+of downloaded files.
+`
+
+var rootChecksumCmd = &cobra.Command{
+	Use:   "checksum",
+	Short: "Checksum of the downloaded files.",
+	Long:  checksumHelper,
+	RunE: func(_ *cobra.Command, _ []string) error {
+		return fmt.Errorf("error, you must also specify a checksum command, like create or check")
```

Comment:
Não tem um jeito melhor de fazer isso? Nunca fiz subcomandos no Cobra. 

Se não tiver, precisamos melhorar essa mensagem. Não precisamos dizer erro, e precisamos instruir as pessoas a utilizar da maneira correta de forma precisa.

```suggestion
		return fmt.Errorf("this command requires a subcommand; options are `create` or `check`")
```

Reply:
Descobri que se omitirmos o `RunE` ele usa uma expressão default que orienta o usuário a utilizar os subcomandos, exibindo as opções. Vou manter assim.

Reply:
Perfeito, tb não conhecia de omitir o `Run`/`RunE`, boa : )

TIL 💜